osgp/protocol-adapter-dlms/osgp-protocol-adapter-dlms/src/main/java/org/opensmartgridplatform/adapter/protocol/dlms/application/services/InstallationService.java

```diff
@@ -49,7 +51,9 @@
 
   @Autowired private InstallationMapper installationMapper;
 
-  @Autowired private EncryptionHelperService encryptionService;
+  @Autowired
+  @Qualifier("decrypterForGxfSmartMetering")
+  private RsaEncrypter rsaEncrypter;
```

Comment:
Wouldn't it make the code more easily understandable if `rsaEncrypter` would get the more descriptive name used in the qualifier?

Reply:
Yes, I changed it